test/integration/testManualConnection.js

```diff
@@ -72,6 +74,41 @@ if (process.env.RUN_MANUAL_TESTS_ONLY === 'true') {
       });
     });
 
+    describe('Connection - ID Token authenticator', async function (done) {
+      const connectionOption = connOption.externalBrowser;
+      connectionOption.clientStoreTemporaryCredential = true;
```

Comment:
It shouldn't be extended this way. It affected more tests. Please fix it by 
'const connectionOption = {
        ...connOption.externalBrowser,
        clientStoreTemporaryCredential: true
      }'

Reply:
Fixed.